VideoWeb/VideoWeb/ClientApp/src/app/services/user-media-stream.service.ts

```diff
@@ -85,6 +85,12 @@ export class UserMediaStreamService {
         this.activeMicrophoneStreamSubject.next(stream);
     }
 
+    closeCurrentStream() {
```

Comment:
could we add a test for this?

Reply:
we can, I've had some real double mocking MediaTracks and it just fails with spys. I;ll have another crack at it but I faced a similar issue in my other or with the v2 media stream service

Reply:
don't worry about it if it's going to be a pain, happy to approve for now so we can get it over the line and address later